packages/www/components/Dashboard/AssetsTable/index.tsx

```diff
@@ -1,23 +1,186 @@
+import { useCallback, useMemo, useState } from "react";
 import Link from "next/link";
-import { ArrowRightIcon } from "@radix-ui/react-icons";
+import { ArrowRightIcon, PlusIcon, Cross1Icon } from "@radix-ui/react-icons";
+import { useApi } from "hooks";
+import Table, { useTableState, Fetcher } from "components/Dashboard/Table";
+import { FilterItem } from "components/Dashboard/Table/filters";
+import TextCell, { TextCellProps } from "components/Dashboard/Table/cells/text";
+import DateCell, { DateCellProps } from "components/Dashboard/Table/cells/date";
+import { SortTypeArgs } from "components/Dashboard/Table/types";
+import { dateSort, stringSort } from "components/Dashboard/Table/sorts";
+import { Column } from "react-table";
+import {
+  CellComponentProps,
+  TableData,
+} from "components/Dashboard/Table/types";
 import {
   Flex,
   Heading,
   Link as A,
+  Button,
   Text,
   Box,
+  AlertDialog,
+  AlertDialogTitle,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogCancel,
+  AlertDialogAction,
+  useSnackbar,
 } from "@livepeer.com/design-system";
+import Spinner from "components/Dashboard/Spinner";
+import { useToggleState } from "hooks/use-toggle-state";
+import CreateAssetDialog from "./CreateAssetDialog";
+
+const filterItems: FilterItem[] = [
+  { label: "Name", id: "name", type: "text" },
```

Comment:
I think in the wireframes there were some additional fields like `source`, `playbackUrl`.  
  
About source, the problem is that this kind of information is not inside the asset, but inside the task that generated it.  
When I was trying to do this frontend in February I tried [this](https://github.com/livepeer/livepeer-com/blob/dc569e2114b7526ede598ab8d218721d2be7215e/packages/www/components/Dashboard/AssetsTable/index.tsx#L215), getting the tasks by userId and grouping the assets by task, getting the type as `source`.  
I think there could be a better way, but in any case the information for now it's there.  
After that I also group the assets by parent assets because at first we wanted to display the children of an asset in an dropdown row, but I think we decided to display all the assets on the same level.
  
About playbackUrl, it's not implemented yet.